MyApp/app/src/main/java/com/example/janari/myapp/EnterIncomeAndExpenses.java

```diff
@@ -0,0 +1,53 @@
+package com.example.janari.myapp;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+
+import static java.lang.Math.round;
+
+
+public class EnterIncomeAndExpenses extends AppCompatActivity {
```

Comment:
I found [Android Activity naming conventions](https://stackoverflow.com/a/38335285) which recommends that generally the best practice is to add "component type" suffix. 

This should* be `EnterIncomeAndExpensesActivity`

\* this decision is yours to make. Also maybe worth documenting in the final paper